src/main/java/marquez/api/models/DatasourcesResponse.java

```diff
@@ -20,5 +20,5 @@
 
 @Value
 public final class DatasourcesResponse {
```

Comment:
_Minor:_ You can also remove `final` from the class definition:

```java
public class DatasourcesResponse {
  ...
}
```

Reply:
sure

Reply:
done